core/src/main/java/com/linecorp/armeria/server/annotation/ExceptionHandlerFunction.java

```diff
@@ -39,12 +39,12 @@
     /**
      * Returns whether the specified {@code cause} is acceptable.
      */
-    default boolean accept(Throwable cause) {
+    default boolean acceptableException(Throwable cause) {
```

Comment:
How about just `canHandleException`?

Reply:
Ok, `can*` would be better for consistency. :-)